console/src/main/scripts/plugins/metrics/ts/alertsCenterList.ts

```diff
@@ -57,16 +57,9 @@ module HawkularMetrics {
                 private $location:ng.ILocationService) {
       $scope.ac = this;
 
-      if ($routeParams.timeOffset === undefined) {
-        $routeParams.timeOffset = 3600000 * 12;
-        $scope.hkParams.timeOffset = $routeParams.timeOffset;
-      }
-
-      this.alertsTimeOffset = $routeParams.timeOffset;
-
       // If the end time is not specified in URL use current time as end time
       this.alertsTimeEnd = $routeParams.endTime ? $routeParams.endTime : Date.now();
-      this.alertsTimeStart = this.alertsTimeEnd - this.alertsTimeOffset;
+      this.alertsTimeStart = this.alertsTimeEnd - HawkularNav.getTimeOffset();
```

Comment:
Point is that if we don't have state, url should be rewritten with the timeoffset parameters.
Or perhaps I miss some alternative ?